src/style.css

```diff
@@ -0,0 +1,26 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  width: 96px;
+  display: flex;
+  justify-content: space-between;
+}
+
+.stars__star {
+  background-image: url("./images/star.svg");
+}
+
+.stars__star2 {
+  background-image: url("./images/star-active.svg");
```

Comment:
Use class-modifier and [nth-child](https://developer.mozilla.org/en-US/docs/Web/CSS/:nth-child) rule to set star-active image